src/TodoForm.jsx

```diff
@@ -1,27 +1,24 @@
 import { useRef, useState } from 'react';
+import TextInputWithLabel from './assets/shared/TextInputWithLabel';
 
 function TodoForm({ onAddTodo }) {
   const [workingTodo, setWorkingTodo] = useState('');
   const todoTitleInput = useRef('');
 
   const handleSubmit = (e) => {
     e.preventDefault();
-    // const title = e.target.title.value;
-    // e.target.title.value = '';
     todoTitleInput.current.focus();
     onAddTodo(workingTodo);
     setWorkingTodo('');
   };
   return (
     <form onSubmit={handleSubmit}>
-      <label htmlFor="todoTitle">Todo: </label>
-      <input
-        id="todoTitle"
-        type="text"
-        name="title"
+      <TextInputWithLabel
         ref={todoTitleInput}
+        elementId="todoTitle"
```

Comment:
Although this works it's best to follow the pattern set by your other prop values. Conventionally you want to pass prop values in `{}`. `elementId={"todoTitle"}`